pyseed/seedclient.py

```diff
@@ -25,31 +25,43 @@
 # Imports from Standard Library
 import inspect
 
-# Imports from Third Party Modules
-import requests
-
 # Local Imports
+import requests
```

Comment:
@nllong This shouldn't be under `Local Imports`, right?

Reply:
This is automated by `isort` so I'm going to leave it.